tests/integration/Owncloud/OcisSdkPhp/OcisTest.php

```diff
@@ -135,4 +138,99 @@ public function testCreateDriveInvalidQuota(int $quota): void
         // no new drive should have been created
         $this->assertCount($countDrivesAtStart, $ocis->listMyDrives());
     }
+
+    public function testListGroups(): void
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $groups = $ocis->listGroups();
+        $this->assertCount(self::GROUP_COUNT, $groups);
+        $this->assertContainsOnlyInstancesOf(Group::class, $groups);
+        foreach ($groups as $group) {
+            $this->assertIsString($group->getId());
+            $this->assertIsString($group->getDisplayName());
+            $this->assertIsArray($group->getGroupTypes());
+            $this->assertIsArray($group->getMembers());
+        }
+    }
+    /**
+     * @return array<int,array<int,string>>
+     */
+    public function expandData(): array
+    {
+        return [
+            ["users"],
+        ];
+    }
+    /**
+     * @dataProvider expandData
+     *
+     * @return void
+     */
+    public function testListGroupsExpanded()
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $groups = $ocis->listGroups(expandMembers: true);
+        $this->assertCount(self::GROUP_COUNT, $groups);
+        foreach ($groups as $group) {
+            $this->assertGreaterThan(0, count($group->getMembers()));
+        }
+    }
+    /**
+     * @return array<int,array<int,string|int>>
+     */
+    public function searchText(): array
+    {
+        return [
+            ["asdas", 0, ""],
```

Comment:
so this tests will not find any group, can we add one that does?
```suggestion
            ["lovers", 0, ["violin-lovers, "physics-lovers"]],
```